specification.md

```diff
@@ -3797,7 +3865,7 @@ You can also use the filter property to filter the state data after data is inje
       address: 1234 SomeStreet
       age: 30
   stateDataFilter:
-    output: "${ {people: [.people[] | select(.age < 40)]} }"
+    output: "${people: [.people[] | select(.age < 40)]}"
```

Comment:
invalid expression

Reply:
thx, same as the other examples that got reduced once too often. will fix